### DISCUSSION TITLE
Failed to install MSI on Windows 10（10.0.18363）

### DISCUSSION THREAD

**Discussion by hhhhskfk** [dokan-dev/dokany#1224] [Q&A] [answered]
Failed to install the tightvnc-2.8.27-gpl-setup-64bit.msi installation package using mirror disk mapping, with the error message 'The system cannot open the specified device or file.' Installation is successful on Win11.
cmd command is: mirror.exe /r d: /l G:
![image](https://github.com/dokan-dev/dokany/assets/19182378/9147ac05-fc1f-4a27-9ac0-b252ec21db0a)

[tightvnc-2.8.27-gpl-setup-64bit.zip](https://github.com/user-attachments/files/15786013/tightvnc-2.8.27-gpl-setup-64bit.zip)

Environment:
Dokan Version: 2.1.0
Operating System: Win10（10.0.18363）

**Comment by Liryna** [comment, marked as answer]
Please try with the `/o` option -> `mirror.exe /r d: /l G: /o`

**Reply by hhhhskfk** [reply]
Thanks, it‘s useful
